api/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+//! # Offline license rs
+//!
+//! **Offline license RS** license generator used for offline software license verification.
+
+pub use crate::license_key::LicenseKey;
+
+pub mod adler32;
```

Comment:
Does the user actually need such module?

Reply:
most of them YES